### DISCUSSION TITLE
Add "Combine internal perimeters every ... layer"

### DISCUSSION THREAD

**Discussion by Bumblebiber** [supermerill/SuperSlicer#1763] [Community contribution]
Hi again,

just like the very useful option "Combine Infille every ... layer", wouldn't it be awesome to have the same one for internal perimeters?

So you could print outer walls very detailed, but the inner ones much faster. Of course, this would only work for certain angles, maybe even only the ones going up straight 90°. But for those prints this would be superior!

**Comment by Mysli0210** [comment]
I would love this feature aswell.
you can kinda trick the slicer into doing so by setting concentric infill to 1% and having an infinitely long anchor while setting infill every n layers to 2.
But having this setting would enable you to still have regular infill.
A bonus of having this could potentially be having stonger layers, as you have 2 outer perimeters bonded to one internally, which in turn is also bonded to the ones below and above.
